GitUpKit/Core/GCRepository+HEAD.m

```diff
@@ -116,7 +116,8 @@ - (GCCommit*)createCommitFromHEADAndOtherParent:(GCCommit*)parent withMessage:(N
   }
 
   const git_commit* parents[2] = {headCommit, parent.private};
-  commit = [self createCommitFromIndex:index withParents:parents count:(headCommit ? (parent ? 2 : 1) : 0)author:NULL message:message error:error];
+  BOOL shouldSign = YES;
```

Comment:
Hi! Looks like this param becomes obsolete? :) [ as down below the call there's reading from prefs ]

Reply:
Nice catch, @osxd!
That's right - this option is obsolete and was used before config read was made. Will fix it soon 😉

Reply:
Fixed ✅